### ISSUE TITLE
Issue deleting labels with "#" in them

### ISSUE THREAD

**Issue by himynameisdave** [jasonbellamy/git-label#15]
I was just trying to delete a label in a repo which happened to have a `#` symbol in the label name. Every time I try to remove these labels, `git-label` announces that it was successful but the items are not actually removed.

**Comment by dilantha111** [comment]
@himynameisdave are you working on this ? If not I can take a look. I am using git-label for a vscode extension that I created. So I am more than happy to give this bug a try.

**Comment by himynameisdave** [comment]
@dilantha111 no, I am not actively working on this, so feel free to go for it.
